dora/core/common/pom.xml

```diff
@@ -135,6 +135,10 @@
       <groupId>org.rocksdb</groupId>
       <artifactId>rocksdbjni</artifactId>
     </dependency>
+    <dependency>
```

Comment:
add dependencies in alphabetically sorted order (as much as possible)

Reply:
done